src/app/audiologist-navigation/audiologist-navigation.component.ts

```diff
@@ -17,57 +16,61 @@ import { Utilities } from '../common/utlilities';
  * active: boolean is a local variable will be switch between true and false to trigger the function.
  */
 export class AudiologistNavigationComponent {
-    @ViewChild(AudiologistSummaryComponent) summaryComponent : AudiologistSummaryComponent;
+  get TabsEnum() { return TabsEnum; }
+  public patientID: string = Utilities.getSessionStorage('patient-id');
+  public active: boolean = true;
+  public scale: number = 0.55;
+  public state: State = new State();
+  @ViewChild(AudiologistSummaryComponent) private summaryComponent: AudiologistSummaryComponent;
 
-    public active: boolean = true;
-    public scale: number = 0.55;
-    public recommendedTests: boolean = false;
-    public suggestedTests: boolean = false;
-    public summary: boolean = true;
-    public notes: boolean = false;
+  constructor(private router: Router) {
+  }
 
-    public onToggle() {
-      if (!this.active) {
-          this.active = true;
-          console.log('is active');
-      } else {
-        this.active = false;
-        console.log('is active');
-      }
+  public onToggle() {
+    if (!this.active) {
+      this.active = true;
+      console.log('is active');
+    } else {
+      this.active = false;
+      console.log('is active');
```

Comment:
Misleading message?

Reply:
Yes, onToggle()  was from the original code, I didn't touch it.